examples/python/pokemon_service.py

```diff
@@ -176,8 +176,8 @@ def get_random_radio_stream(self) -> str:
 @app.middleware
 async def check_content_type_header(request: Request, next: Next) -> Response:
     content_type = request.headers.get("content-type")
-    if content_type == "application/json":
-        logging.debug("found valid `application/json` content type")
+    if content_type in ["application/json", "application/vnd.amazon.eventstream"]:
+        logging.debug("found valid `%s` content type", content_type)
     else:
         logging.warning(
```

Comment:
nit: Should we add a comment here indicating that sensitive information may be logged by dumping all headers and that this should not be done in production?

Reply:
Sorry, I had "add to merge queue after approval" enabled and the PR got merged before I saw this comment.

I've addressed it in https://github.com/awslabs/smithy-rs/pull/3083.